contracts/ESN/RentingDapp/ProductManager.sol

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+import { RentalAgreement } from "./RentalAgreement.sol";
+
+contract ProductManager
+{
```

Comment:
@mridul4101 You will need to run prettier, `yarn style-fix` should do it for you